### PR TITLE
Release v1.0.0

### PR DESCRIPTION
**Pull request by oscar-nutonomy**
Base-branch for release v1.0.0. TODOs:

- [x] Updated README and install instructions.
- [x] Move setup files to `setup` folder. [CL]
- [x] Update README for debris [OB]
- [x] Filter on bike-racks during eval. [SV]
- [x] Simple eval tests. [VB]
- [x] Update eval renderers. [VB]
- [x] Add back the CLI. [VB]
- [x] Test TP metrics. [VB]
- [x] Update visualize_sample to operate on new structs. [VB]
- [x] [create_split_scenes()](https://github.com/nutonomy/nuscenes-devkit/blob/release_v0.2/python-sdk/nuscenes/utils/splits.py#L190) currently gives access to `all` split. It shouldn't be allowed.
---- OPTIONAL ----
- [ ] Fix `quaternion_yaw` method to be valid for all frames. Or make frame req. more explicit. [OB]
- [ ] Look into adding box velocity by default. [OB]
- [ ] More tests. [VB]
- [ ] Unit-tests for orientation evaluation. [VB]
- [ ] Either make evaluation rendering independent of rest of eval code or add deserialize to DetectionMetrics

### CODE REVIEW

File: python-sdk/tutorial.ipynb [hunk 1]
```diff
@@ -8,22 +8,20 @@
...
-   },
+   "metadata": {},
    "outputs": [],
    "source": [
     "# Let's start by initializing the database\n",
     "%matplotlib inline\n",
     "from nuscenes.nuscenes import NuScenes\n",
     "\n",
-    "nusc = NuScenes(version='v0.1', dataroot='/data/nuscenes', verbose=True)"
+    "nusc = NuScenes(version='v1.0', dataroot='/data/sets/nuscenes', verbose=True)"
```
Comment (Qiang-Xu) [inline python-sdk/tutorial.ipynb]: `v1.0-mini`?

Comment (oscar-nutonomy) [reply python-sdk/tutorial.ipynb]: Yes please. That one reverted back somehow.

Comment (Alex-nutonomy) [reply python-sdk/tutorial.ipynb]: `v1.0-mini` would be the best default.

**Comment by bassam-nutonomy** [comment]
I made the following changes to the README:

- confirm to the terms of service -> agree to the terms of service

- If  you want to use another folder, specify the dataroot parameter of the NuScenes class below. -> nuscenes isn't mentioned below so I said: dataroot parameter of the NuScenes class (see tutorial) 

- There is a broken link to advanced installation instructions. Sourabh is looking into it.


Changes to schema.md:

- Taxonomy of object categories (e.g. vehicle, human). Subcategories are delineated by a period -> Taxonomy of object categories (e.g. vehicle, human). Subcategories are delineated by a period (e.g. human.pedestrian.adult).

**Pull request merged**